src/transforms/stdnames.jl

```diff
@@ -0,0 +1,59 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+"""
+    StdNames(S)
+
+StdNames transform standardizes the column names.
+"""
+struct StdNames <: Stateless
+    stdmethod::Symbol
+end
+
+isrevertible(::Type{StdNames}) = true
+
+function apply(transform::StdNames, table)    
+    if transform.stdmethod == :camel
+        _camel(table)
+    elseif transform.stdmethod == :snake
+        _snake(table)
+    else
+        _upper(table)
+    end
```

Comment:
You can use the short cut:

```julia
spec = transform.spec

spec == :camel && _camel(table)
spec == :snake && _snake(table)
spec == :upper && _upper(table)
```

Reply:
This doesn't work as expected. It works fine while `spec = :upper`, but it returns `false` otherwise.

Reply:
I may have missed parenthesis. What about `(spec == :camel) && _camel(table)`? Anyways, I think you will need to refactor the logic a bit more to move more lines of code to apply and refactor _camel, _snake and _upper to take strings as input.